spec/operators/bufferToggle-spec.ts

```diff
@@ -341,4 +342,54 @@ describe('Observable.prototype.bufferToggle', () => {
     expectSubscriptions(e1.subscriptions).toBe(e1subs);
     expectSubscriptions(e2.subscriptions).toBe(e2subs);
   });
+
+  it('should accept closing selector returns promise resolves', (done: DoneSignature) => {
```

Comment:
I would add the word `that` here. "should accept closing selector **that** returns **a resolved** promise"